<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Addtc</class>
 <widget class="QDialog" name="Addtc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEdit_7">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
  background-color: #ffffff; 
  color: rgb(114, 159, 207);
  border-style: inherit;
  font-family:Verdana;
font-size:10;
font-weight:bold;
border-width: 2px;
border-style:solid;
border-color:rgb(114, 159, 207);
border-radius:8px;
}

QLineEdit:hover{
   
border-color: rgb(179, 119, 0);
}

</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
  background-color: #ffffff; 
  color: rgb(114, 159, 207);
  border-style: inherit;
  font-family:Verdana;
font-size:10;
font-weight:bold;
border-width: 2px;
border-style:solid;
border-color:rgb(114, 159, 207);
border-radius:8px;
}

QLineEdit:hover{
   
border-color: rgb(179, 119, 0);
}

</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
  background-color: #ffffff; 
  color: rgb(114, 159, 207);
  border-style: inherit;
  font-family:Verdana;
font-size:10;
font-weight:bold;
border-width: 2px;
border-style:solid;
border-color:rgb(114, 159, 207);
border-radius:8px;
}

QLineEdit:hover{
   
border-color: rgb(179, 119, 0);
}

</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Добавить владельца?</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
  background-color: #ffffff; 
  color: rgb(114, 159, 207);
  border-style: inherit;
  font-family:Verdana;
font-size:10;
font-weight:bold;
border-width: 2px;
border-style:solid;
border-color:rgb(114, 159, 207);
border-radius:8px;
}

QLineEdit:hover{
   
border-color: rgb(179, 119, 0);
}

</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEdit">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
  background-color: #ffffff; 
  color: rgb(114, 159, 207);
  border-style: inherit;
  font-family:Verdana;
font-size:10;
font-weight:bold;
border-width: 2px;
border-style:solid;
border-color:rgb(114, 159, 207);
border-radius:8px;
}

QLineEdit:hover{
   
border-color: rgb(179, 119, 0);
}

</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
  background-color: #ffffff; 
  color: rgb(114, 159, 207);
  border-style: inherit;
  font-family:Verdana;
font-size:10;
font-weight:bold;
border-width: 2px;
border-style:solid;
border-color:rgb(114, 159, 207);
border-radius:8px;
}

QLineEdit:hover{
   
border-color: rgb(179, 119, 0);
}

</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>10000</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(114, 159, 207);
border:solid rgb(114, 159, 207);
border-radius:6px;
}

QPushButton:hover{
background-color: rgb(179, 119, 0);
border:solid rgb(179, 119, 0);
border-radius:4px;
}
</string>
     </property>
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>10000</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(114, 159, 207);
border:solid rgb(114, 159, 207);
border-radius:6px;
}

QPushButton:hover{
background-color: rgb(179, 119, 0);
border:solid rgb(179, 119, 0);
border-radius:4px;
}
</string>
     </property>
     <property name="text">
      <string>ОК</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_6</tabstop>
  <tabstop>lineEdit_7</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>lineEdit_5</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
